### ISSUE TITLE
Hyperparameter Configuration and Difficulty level

### ISSUE THREAD

**Issue by glenbhermon** [hukaixuan19970627/yolov5_obb#501]
1. I am working on object detection using aerial images 
2. Developed the code using the described process but the results are not promising.
3. Few queries are requested to be clarified:
   a) What is the 'difficult' parameter in the annotation format? And how does it affect performance?
   b) What should be the minimum number of images used to train each class?
   c) Within the hyperparemeter 'yaml file', how do we choose each parameter for better training?

![Result](https://user-images.githubusercontent.com/38610046/203784632-e1555a4a-7ff4-4466-ad54-7d2b47eaf194.png)

Please Help! Thanks in advance.

**Comment by glenbhermon** [comment]
The "hyp.finetune_dota_CloseAug.yaml" file seems to work better than the "hyp.finetune_dota.yaml" file.
Please Help!

**Comment by checker999** [comment]
During the training process, the calibration object with difficult=1 will be ignored directly

**Comment by davidsvy** [comment]
> During the training process, the calibration object with difficult=1 will be ignored directly

This is incorrect. Objects with difficult=2 are ignored, as evidenced [here](https://github.com/hukaixuan19970627/yolov5_obb/blob/master/utils/datasets.py#L980):

```python
if label[-1] == "2": # diffcult
    continue
```
